### PR TITLE
get country by ioc

### PR DESCRIPTION
**Pull request by roquib**
This PR adds a function where we can get country config by country ioc

### CODE REVIEW

File: src/countries/index.test.ts [hunk 1]
```diff
@@ -29,6 +29,11 @@ describe('country helpers', () => {
       expect(getCountryByName('Russia')).toMatchSnapshot();
       expect(getCountryByName('Palestine')).toMatchSnapshot();
     });
+
+    it('returns country info by ioc', () => {
```
Comment (thehungrycoder) [inline src/countries/index.test.ts:33]: what is ioc?

Comment (thehungrycoder) [reply src/countries/index.test.ts:33]: i'm not sure why we need this `ioc`. what's the difference between this and the short name?

Comment (roquib) [reply src/countries/index.test.ts:33]: IOC is the official country code of the International Olympic Committee. You can find this on Wikipedia [Country IOC](https://en.wikipedia.org/wiki/List_of_IOC_country_codes).  You can find this on every country key here `src/countries/list.ts` file.
### Sample Ioc code 
| Country      | IOC |
| ----------- | ----------- |
| United States      | USA       |
| Great Britain   | GBR        |
| Australia   | AUS        |

if a country name contains this IOC short name we can find the config easily.

Comment (thehungrycoder) [reply src/countries/index.test.ts:33]: Isn't it same as Short code? do we need this?

Comment (thehungrycoder) [reply src/countries/index.test.ts:33]: This list is hard to maintain. To me it seems short code and this is same. Hence, we've two options

1) We can use short code. This is already implemented
2) In mobile project, we can determine Full Country Name and use that.

**Comment by roquib** [comment]
I am closing this PR  for maintenance.

**Pull request closed without merging**